cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss

```diff
@@ -22,6 +22,22 @@
     max-height: 550px;
     .grid-item {
       grid-template-columns: 20px 1.5fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 20px;
+
+      .sortable-header {
+        cursor: pointer;
+        display: flex;
+        align-items: center;
+        text-align: right;
```

Comment:
any specific reason why this is align right? I am right now seeing this

![screen shot 2018-04-03 at 4 51 12 pm](https://user-images.githubusercontent.com/1452845/38281383-424863b4-375f-11e8-9907-cff4819e906c.png)

vs
 
![screen shot 2018-04-03 at 4 51 18 pm](https://user-images.githubusercontent.com/1452845/38281382-4222ac00-375f-11e8-8217-5bea01affb99.png)

Reply:
No reason, I didn't look at the mocks correctly